packaging/requirements/requirements-azure.txt

```diff
@@ -16,4 +16,4 @@ azure-mgmt-web>=0.32.0,<0.33
 azure-mgmt-containerservice>=3.0.1
 azure-mgmt-containerregistry>=1.0.1
 azure-mgmt-rdbms>=0.2.0rc1,<0.3.0
-azure-mgmt-containerinstance>=0.3.1,<0.4.0
+azure-mgmt-containerinstance==0.4.0
```

Comment:
fully tested?

Reply:
yes, went through all the tests locally and globally